apps/darts-modernisation/ithc/base/darts-api-function.enc.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+data:
+  DARTS_API_DB_CONNECTION_STRING: ENC[AES256_GCM,data:YIK4EY0SqJZbV2YWFvnADZzgd+nUKsPX3mvlNVzAM5oUL1qiDzxWuNR+SCOQQmAySguRvDOuagLpvhHYf6AZpgxru0tLFJ8IQ1Pxy081gEaA5+fdzlEfZpdO9uaJrAv/RVElevAahE7HlePuF1D/xZP95FQrHLq3VFy6qA==,iv:16okVrHh5ewjqyn+oNIHrYNHIc9tqAxuvgauoIa46pY=,tag:8E03qjOE9JRZmFiYffW+xw==,type:str]
```

Comment:
I think this need to be re-encrypted with sops, it can't be copied from staging as it's a different value and a different key